lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -18,10 +18,11 @@ def self.included(base)
 
   public
   def setup_jdbc_config
-    # JDBC driver library path to third party driver library.
+    # JDBC driver library path to third party driver library. In case of multiple libraries being
+    # required you can pass them separated by a comma.
     #
     # If not provided, Plugin will look for the driver class in the Logstash Java classpath.
-    config :jdbc_driver_library, :validate => :path
+    config :jdbc_driver_library, :validate => :string
```

Comment:
It will be, I agree, but then this changes the configuration making all previous ones incompatible. Is this something we want to make?